morphocut/image.py

```diff
@@ -121,7 +121,7 @@ def transform_stream(self, stream):
                     sl = self._enlarge_slice(sl, self.padding)
 
                 props = skimage.measure._regionprops._RegionProperties(
-                    sl, i + 1, labels, image, True, "rc"
+                    sl, i + 1, labels, image, True
```

Comment:
skimage.measure is still giving error saying that the constructor accepts less arguments, and when I removed the key "rc", it started working fine. Kindly review the change

Reply:
which version of skimage are your using?

Reply:
0.16.1

Reply:
Then please remove the "rc" as it was deprecated in 0.16.0. I set the version accordingly in setup.py.